pkg/controller/gitopsconfig/controller.go

```diff
@@ -44,6 +45,14 @@ import (
 )
 
 var log = logf.Log.WithName("controller_gitopsconfig")
+var runningJobMap = make(map[string]bool)
```

Comment:
Adding a map that is accessed from multiple goroutines adds a dangerous [data race](https://en.wikipedia.org/wiki/Data_race#Software). Please at least change this to a [sync.Map](https://golang.org/pkg/sync/#Map).

Reply:
done